### DISCUSSION TITLE
Coder on Docker

### DISCUSSION THREAD

**Discussion by Vichet97** [coder/code-server#5639] [Q&A]
How do i use coder cli when the Coder is only installed in Docker ? The getting started shows the usage of it but not when it's inside docker

**Comment by lg-nineoneninethreenine** [comment]
Can you share the link for the getting started guide you visited?

**Comment by Vichet97** [comment]
https://coder.com/docs/coder-oss/latest/quickstart/docker

Here it just have the installation page then Quickstart page where it tells
to use coder cli. But to to use coder cli when it's inside docker

**Reply by lg-nineoneninethreenine** [reply]
You mean to say that you use Coder in a Docker environment?

**Comment by Vichet97** [comment]
Yes, that's right.
